src/components/Main/Main.jsx

```diff
@@ -5,6 +5,7 @@ import AddTask from '../AddTask/AddTask';
 import TaskList from '../TaskList/TaskList';
 
 export default function Main() {
+    const [tasks, setTasks] = React.useState([]);
```

Comment:
Keep in mind that initiating state like this will require you to "prop drill" down to whatever components need access to this piece of state, this is typically fine if you're only doing it for like 1-3 files, but just keep in mind that with larger apps, this would cause chaos within the codebase, e.g if someone needs to find out where this specific piece of state is coming from, they would need to manually trace back to this file.

Reply:
Side note, to remedy this, look up React Context(built-in) and Redux(Library)